tests/LINEBot/ManagementAudienceTest.php

```diff
@@ -0,0 +1,476 @@
+        $this->assertEquals('READY', $res->getJSONDecodedBody()['status']);
+        $this->assertEquals(1000, $res->getJSONDecodedBody()['audienceCount']);
+        $this->assertEquals(1500351844, $res->getJSONDecodedBody()['created']);
+        $this->assertEquals('f70dd685-499a-4231-a441-f24b8d4fba21', $res->getJSONDecodedBody()['requestId']);
+        $this->assertEquals('https://line.me/en', $res->getJSONDecodedBody()['clickUrl']);
+        $this->assertEquals(false, $res->getJSONDecodedBody()['isIfaAudience']);
+        $this->assertEquals('READ', $res->getJSONDecodedBody()['permission']);
+        $this->assertEquals('MESSAGING_API', $res->getJSONDecodedBody()['createRoute']);
+        $this->assertEquals([
+            [
+                'audienceGroupJobId' => 8389303728991,
+                'audienceGroupId' => 4389303728991,
+                'description' => 'JOB1 DESCRIPTION',
+                'type' => 'DIFF_ADD',
+                'jobStatus' => 'WORKING',
+                'audienceCount' => 500,
+                'created' => 1500351944,
+            ], [
+                'audienceGroupJobId' => 8389303728992,
+                'audienceGroupId' => 4389303728991,
+                'description' => 'JOB2 DESCRIPTION',
+                'type' => 'DIFF_ADD',
+                'jobStatus' => 'FAILED',
+                'failedType' => 'INTERNAL_ERROR',
+                'audienceCount' => 500,
+                'created' => 1500351944,
+            ],
+        ], $res->getJSONDecodedBody()['jobs']);
+    }
+
+    public function testGetAuthorityLevel200()
+    {
+        $mock = function ($testRunner, $httpMethod, $url, $data) {
+            /** @var \PHPUnit\Framework\TestCase $testRunner */
+            $testRunner->assertEquals('GET', $httpMethod);
+            $testRunner->assertEquals('https://api.line.me/v2/bot/audienceGroup/authorityLevel', $url);
+
+            return [
+                'authorityLevel' => 'PUBLIC',
+            ];
+        };
+        $bot = new LINEBot(new DummyHttpClient($this, $mock), ['channelSecret' => 'CHANNEL-SECRET']);
+        $res = $bot->getAuthorityLevel();
+
+        $this->assertEquals(200, $res->getHTTPStatus());
+        $this->assertTrue($res->isSucceeded());
+        $this->assertEquals('PUBLIC', $res->getJSONDecodedBody()['authorityLevel']);
+    }
+
+    public function testUpdateAuthorityLevel200()
+    {
+        $mock = function ($testRunner, $httpMethod, $url, $data) {
+            /** @var \PHPUnit\Framework\TestCase $testRunner */
+            $testRunner->assertEquals('GET', $httpMethod);
+            $testRunner->assertEquals('https://api.line.me/v2/bot/audienceGroup/authorityLevel', $url);
+
+            return [
+                'authorityLevel' => 'PUBLIC',
+            ];
+        };
+        $bot = new LINEBot(new DummyHttpClient($this, $mock), ['channelSecret' => 'CHANNEL-SECRET']);
+        $res = $bot->getAuthorityLevel();
```

Comment:
Is it updateAuthorityLevel instead of getAuthorityLevel?

Reply:
yes. fixed~